test/testutils.jl

```diff
@@ -0,0 +1,330 @@
+
+struct ImputorTester{I<:Imputor}
+    imp::Type{I}
+    f::Function
+    f!::Function
+    kwargs::Dict
+end
+
+function ImputorTester(imp::Type{<:Imputor}; kwargs...)
```

Comment:
Can we have a docstring listing all the Kwargs?

Reply:
The issue is that the kwargs will vary based on the Imputor type.